tests/test_watermark_no_torch_compile_1615.py

```
@@ -0,0 +1,130 @@
+
+from services import watermark
+from services.watermark import detect_watermark, embed_watermark
+
+SR = 24000
+
+
+def _moshi_compile_module():
+    """AudioSeal's vendored moshi compile switch, or None when unavailable."""
+    try:
+        from audioseal.libs.moshi.utils import compile as moshi_compile
+    except Exception:  # pragma: no cover — exercised via the skip below
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)